reftests/src/helpers.rs

```diff
@@ -169,11 +191,16 @@ pub fn generate_key(key_seed: u8) -> (ed25519_dalek::Keypair, Vec<u8>, Header) {
 
 /// Create an envelope with a message and an optional key_seed. If the key_seed is
 /// specified, the protected headers and signatures will be filled.
-pub fn envelope<M: AsRef<[u8]>>(key_seed: Option<u8>, message: M) -> CoseSign1 {
+pub fn envelope<M: AsRef<[u8]>>(
```

Comment:
Refactor, DRY

Reply:
I think envelope is ok. How would you refactor it?

Reply:
Refactored